src/main/resources/assets/js/application/services/tdpInvestAuthService.js

```diff
@@ -3,12 +3,24 @@ define(['angular','application/tdpInvestModule', 'application/services/tdpInvest
         var service = {};
 
         service.login = function(username, password) {
-            return $http.post('/api/login', { mail: username, password: password }).then(handleSuccess, handleError('Bad credentials'));
+            service.setCredentials(username, password);
+
+            return $http.get('/api/login').then(function(res) {
+                service.setCredentials($rootScope.globals.currentUser.username, res.data);
```

Comment:
we should not be using globals like these - we will leave it for now - but it's a good example of technical debt